layouts/download-releases.hbs

```diff
@@ -42,6 +42,9 @@
                                         <a href="{{changeloglink version}}">
                                             {{i18n 'links.pages.changelog'}}
                                         </a>
+                                        <a href="{{apidocslink version}}">
+                                            {{i18n 'releases.documentation'}}
```

Comment:
I suggest using `docs.text` instead, for consistency. (defined here: https://github.com/nodejs/new.nodejs.org/pull/161/files#diff-7c141f233ad3e5f2b1dd471e292aec5dR59)